### ISSUE TITLE
Missing complexType "clobType” or "blobType" in table.xsd (SIARD Output)

### ISSUE THREAD

**Issue by Chlara** [keeps/db-preservation-toolkit#33]
If "clobType” or "blobType" are used in the complexType “rowType” then the table.xsd must contain the complexType "clobType” or "blobType".

Wrong / v2.0-rc2:
```
<?xml version="1.0" encoding="UTF-8"?>
<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns="http://www.admin.ch/xmlns/siard/1.0/northwind/suppliers.xsd" attributeFormDefault="unqualified" elementFormDefault="qualified" targetNamespace="http://www.admin.ch/xmlns/siard/1.0/northwind/suppliers.xsd">
	<xs:element name="table">
		<xs:complexType>
			<xs:sequence>
				<xs:element maxOccurs="unbounded" minOccurs="0" name="row" type="rowType">
				</xs:element>
			</xs:sequence>
		</xs:complexType>
	</xs:element>
	<xs:complexType name="rowType">
		<xs:sequence>
			<xs:element name="c1" type="xs:decimal"/>
			<xs:element minOccurs="0" name="c2" type="xs:string"/> 
			<xs:element minOccurs="0" name="c3" type="xs:string"/>
			<xs:element minOccurs="0" name="c4" type="xs:string"/>
			<xs:element minOccurs="0" name="c5" type="xs:string"/>
			<xs:element minOccurs="0" name="c6" type="xs:string"/>
			<xs:element minOccurs="0" name="c7" type="xs:string"/>
			<xs:element minOccurs="0" name="c8" type="xs:string"/>
			<xs:element minOccurs="0" name="c9" type="xs:string"/>
			<xs:element minOccurs="0" name="c10" type="xs:string"/>
			<xs:element minOccurs="0" name="c11" type="xs:string"/>
			<xs:element minOccurs="0" name="c12" type="clobType"/>
		</xs:sequence>
	</xs:complexType>
</xs:schema>
```

Correct:
```
<?xml version="1.0" encoding="UTF-8"?>
<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns="http://www.admin.ch/xmlns/siard/1.0/northwind/suppliers.xsd" attributeFormDefault="unqualified" elementFormDefault="qualified" targetNamespace="http://www.admin.ch/xmlns/siard/1.0/northwind/suppliers.xsd">
	<xs:element name="table">
		<xs:complexType>
			<xs:sequence>
				<xs:element maxOccurs="unbounded" minOccurs="0" name="row" type="rowType">
				</xs:element>
			</xs:sequence>
		</xs:complexType>
	</xs:element>
	<xs:complexType name="rowType">
		<xs:sequence>
			<xs:element name="c1" type="xs:decimal"/>
			<xs:element minOccurs="0" name="c2" type="xs:string"/> 
			<xs:element minOccurs="0" name="c3" type="xs:string"/>
			<xs:element minOccurs="0" name="c4" type="xs:string"/>
			<xs:element minOccurs="0" name="c5" type="xs:string"/>
			<xs:element minOccurs="0" name="c6" type="xs:string"/>
			<xs:element minOccurs="0" name="c7" type="xs:string"/>
			<xs:element minOccurs="0" name="c8" type="xs:string"/>
			<xs:element minOccurs="0" name="c9" type="xs:string"/>
			<xs:element minOccurs="0" name="c10" type="xs:string"/>
			<xs:element minOccurs="0" name="c11" type="xs:string"/>
			<xs:element minOccurs="0" name="c12" type="clobType"/>
		</xs:sequence>
	</xs:complexType>
	<xs:complexType name="clobType">
		<xs:simpleContent>
			<xs:extension base="xs:string">
				<xs:attribute name="file" type="xs:string"/>
				<xs:attribute name="length" type="xs:integer"/>
			</xs:extension>
		</xs:simpleContent>
	</xs:complexType>
	<xs:complexType name="blobType">
		<xs:simpleContent>
			<xs:extension base="xs:hexBinary">
				<xs:attribute name="file" type="xs:string"/>
				<xs:attribute name="length" type="xs:integer"/>
			</xs:extension>
		</xs:simpleContent>
	</xs:complexType>
</xs:schema>
```

**Comment by hsilva-keep** [comment]
Dear @Chlara 
We have fixed that problem precisely in tag v2.0-rc2. Can you please make sure that you're using that same code (see if the following file/line matches your source code: https://github.com/keeps/db-preservation-toolkit/blob/master/src/main/java/pt/gov/dgarq/roda/common/convert/db/modules/siard/out/SIARDExportModule.java#L1116 ; also, please make sure that the version you're using is compiled version with that source code (issue an "mvn clean package" just to make sure)).

If the answer to the previous is no, that is, you are using the latest source code (the one I've mentioned) and still the generated table.xsd is incorrect, please share with us the steps/database you're using so we are able to reproduce it and confirm if an bug still exists.

**Comment by Chlara** [comment]
Dear @hsilva-keep 
My college has tested rc1 :-(
so this Issue is closed.
Chlara
